Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/overlay/InputOverlay.java

```diff
@@ -82,7 +83,7 @@ public InputOverlay(Context context, AttributeSet attrs)
     super(context, attrs);
 
     mPreferences = PreferenceManager.getDefaultSharedPreferences(getContext());
-    if (!mPreferences.getBoolean("OverlayInit", false))
+    if (!mPreferences.getBoolean("OverlayInitV2", false))
```

Comment:
Does this mean that existing layouts are overwritten (even if they were ok for either landscape or portrait before)?

Reply:
No, I put checks in place if they already setup their config. This will only add any overlays they have yet to move. I only upgraded to V2 so that it would now check for the portrait ones.

https://github.com/dolphin-emu/dolphin/blob/8638cee092b099d61e60844c20ed8de64723b88e/Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/overlay/InputOverlay.java#L921